airflow/contrib/operators/bigquery_operator.py

```diff
@@ -76,3 +76,48 @@ def execute(self, context):
         cursor = conn.cursor()
         cursor.run_query(self.bql, self.destination_dataset_table, self.write_disposition,
                          self.allow_large_results, self.udf_config)
+
+class BigQueryDeleteOperator(BaseOperator):
```

Comment:
Could you put this in its own class?

BigQueryTableDeleteOperator

and

bigquery_table_delete_operator.py